src/components/templates/ScreenShare/components/Player/Player.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useRef, useEffect } from "react";
+import classNames from "classnames";
+import {
+  ILocalVideoTrack,
+  IRemoteVideoTrack,
+  ILocalAudioTrack,
+  IRemoteAudioTrack,
+} from "agora-rtc-sdk-ng";
+import "./Player.scss";
+
+import VolumeMutedIcon from "assets/icons/volume-muted-icon.svg";
+import CameraOffIcon from "assets/icons/camera-off-icon.svg";
+import StreamingIcon from "assets/icons/streaming-icon.svg";
+
+export interface VideoPlayerProps {
+  videoTrack: ILocalVideoTrack | IRemoteVideoTrack | undefined;
+  audioTrack?: ILocalAudioTrack | IRemoteAudioTrack | undefined;
+  showButtons?: boolean;
+  isCamOn?: boolean;
+  isMicOn?: boolean;
+  isSharing?: boolean;
+  toggleCam?: () => void;
+  toggleMic?: () => void;
+}
+
+const Player = ({
+  videoTrack,
+  audioTrack,
+  showButtons = false,
+  isCamOn,
+  isMicOn,
+  isSharing,
+  toggleCam,
+  toggleMic,
+}: VideoPlayerProps) => {
+  const container = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!container.current) return;
+    videoTrack?.play(container.current);
+    return () => {
+      videoTrack?.stop();
+    };
+  }, [container, videoTrack]);
+
+  useEffect(() => {
+    audioTrack?.play();
+    return () => {
+      audioTrack?.stop();
+    };
+  }, [audioTrack]);
```

Comment:
spaces after the guard + before the return

Reply:
Should the guard also contain `if (!videoTrack) return` rather than needing to use `videoTrack?.` everywhere?

Same comment for `audioTrack`